README.md

```diff
@@ -120,6 +120,39 @@ _No codes defined as messages are sent by queue_
 
 **Response Body**
 For debugging purposes, the patched Task is returned.
+  
+#### HTTP GET `/discovery/` (on tapas-auction-house)
+
+**Notes**
+- There is no request body
+- The response's task types are the ones that are sought, i.e. that cannot be executed locally (this way, the crawling auction house should subscribe to the discovered auction house if it can offer that service)
+  
+**Media-Type** `application/auctionhousediscovery+json`
+
+**Response Codes**
+
+- `200` (Accepted) The patch was applied to the Task
+
+**Response Body**
+  
+```json
+  {
+  "items": [
+    {
+      "uri":"http://example.org",
```

Comment:
Comment from WebSub expert Erik:
This needs to be the topic URI of the auction house, which can then be used to discover the websubhub.
Just in case we did not dicuss that anyways

Reply:
Added as a note in 0da0ee6303. So can the auction house's URI be resolved using the topic URI?

Reply:
I thought we wanted to have both URIs? The clean one, and the one with the path to the websub? maybe call them auctionhouseuri and websuburi?

Reply:
Jup Kris is right. So we woud just add another uri i guess. Naming is fine for me

Reply:
Both are updated. I'm in lecture now so feel free to update it without me if you don't want to wait for me.